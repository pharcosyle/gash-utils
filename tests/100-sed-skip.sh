input='1
2
3
4
5
6'

echo "$input" | \sed 'n;n;s/./x/'
