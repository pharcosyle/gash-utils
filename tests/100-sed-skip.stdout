1
2
x
4
5
x
